charts/kubeclarity/templates/ingress.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.kubeclarity.ingress.enabled -}}
+{{- $kubeVersion := .Capabilities.KubeVersion.Version }}
+{{- $extraPaths := .Values.kubeclarity.ingress.extraPaths -}}
+{{- $pathType := .Values.kubeclarity.ingress.pathType -}}
+{{- $serviceName := include "kubeclarity.name" . -}}
+{{- $servicePort := .Values.kubeclarity.service.port -}}
+{{ if semverCompare ">= 1.19.0-0" $kubeVersion }}
+apiVersion: networking.k8s.io/v1
+{{ else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
+{{ else }}
+apiVersion: extensions/v1beta1
+{{ end }}
+kind: Ingress
+metadata:
+  name: {{ include "kubeclarity.name" . }}
+  namespace: '{{ .Release.Namespace }}'
+  labels:
+    {{ include "kubeclarity.labels" . }}
+  {{- template "kubeclarity.ingress.annotations" . }}
+spec:
+{{- if .Values.kubeclarity.ingress.enabled }}
```

Comment:
did you mean `{{- if .Values.kubeclarity.ingress.tls }}`

Reply:
That's right, corrected.